server/task_executor_schema_update_ghost_sync.go

```diff
@@ -177,23 +176,22 @@ func executeSync(ctx context.Context, l *zap.Logger, task *api.Task, mi *db.Migr
 		return -1, "", err
 	}
 
-	insertedID, err := util.BeginMigration(ctx, executor, mi, prevSchemaBuf.String(), statement, databaseName)
+	insertedID, err := util.BeginMigration(ctx, executor, mi, prevSchemaBuf.String(), statement, db.BytebaseDatabase)
 	if err != nil {
 		return -1, "", err
 	}
 	startedNs := time.Now().UnixNano()
 
 	defer func() {
-		if err := util.EndMigration(ctx, l, executor, startedNs, insertedID, updatedSchema, databaseName, resErr == nil /*isDone*/); err != nil {
+		if err := util.EndMigration(ctx, l, executor, startedNs, insertedID, updatedSchema, db.BytebaseDatabase, resErr == nil /*isDone*/); err != nil {
```

Comment:
Add a comment on why we use bytebase database instead of databaseName?

Reply:
ditto.

Reply:
duplicated.